client/src/style.css

```diff
@@ -0,0 +1,45 @@
+.homes {
```

Comment:
Is there a component currently receiving this class? I know I don't see all files, so I may have missed it!

Reply:
'Tis not! Good catch...